db/migrate/20230822090606_create_books.rb

```diff
@@ -0,0 +1,11 @@
+class CreateBooks < ActiveRecord::Migration[7.0]
+  def change
+    create_table :books do |t|
+      t.string :name
```

Comment:
we can add constraints directly to a database by adding column modifiers

https://guides.rubyonrails.org/active_record_migrations.html#column-modifiers